malaria-ios/ViewControllers/RapidFireViewController.swift

```diff
@@ -0,0 +1,159 @@
+import UIKit
+
+class RapidFireViewController: UIViewController {
+  
+  @IBOutlet weak var questionLabel: UILabel!
+  @IBOutlet var answerButtons: [UIButton]!
+  @IBOutlet weak var countDownLabel: UILabel!
+  @IBOutlet weak var scoreLabel: UILabel!
+  
+  @IBInspectable let TimerBlinkAtSecond: Int = 3
+  @IBInspectable let TimerMaxValue: Int = 5
+  
+  var rapidFireGame: RapidFireGame?
+  
+  private var timer: NSTimer?
+  
+  private var currentLevel = 0 {
+    didSet {
+      // Check if we ran out of questions
+      if rapidFireGame!.entries.count == currentLevel {
+        endGame()
+        return
+      }
+      
+      // Set the question
+      questionLabel.text = rapidFireGame?.entries[currentLevel].question
+      
+      // Set the answer buttons
+      let answers = rapidFireGame?.entries[currentLevel].answers
+      for (index, answer) in (answers?.enumerate())! {
+        answerButtons[index].setTitle(answer, forState: .Normal)
+      }
+      
+      // Set timer
+      timer = NSTimer.scheduledTimerWithTimeInterval(1, target: self, selector: #selector(updateTimer), userInfo: nil, repeats: true)
```

Comment:
What is happening here? Move the 1 variable to an constant (the name of the contents should make things self-explanatory)